test/model/prophet/test_prophet.py

```diff
@@ -41,17 +50,17 @@ def test_feat_dynamic_real_success():
                 ),
             }
         ],
-        freq="1D",
+        freq=freq,
     )
 
     predictor = ProphetPredictor(**params)
 
     act_fcst = next(predictor.predict(dataset))
     exp_fcst = np.arange(5.0, 5.0 + params["prediction_length"])
 
-    assert np.all(np.isclose(act_fcst.quantile(0.1), exp_fcst, atol=0.02))
-    assert np.all(np.isclose(act_fcst.quantile(0.5), exp_fcst, atol=0.02))
-    assert np.all(np.isclose(act_fcst.quantile(0.9), exp_fcst, atol=0.02))
+    assert exp_fcst.shape == act_fcst.quantile(0.1).shape
+    assert exp_fcst.shape == act_fcst.quantile(0.5).shape
+    assert exp_fcst.shape == act_fcst.quantile(0.9).shape
```

Comment:
The previous assertions were very strict, and some started failing with `prophet==1.1`; I opted for making this test not about Prophet's accuracy, but rather about the wrapper's behavior.

Concerns @jaheba?

Reply:
Interesting that it gets worse. I think this makes sense, as we can't guarantee prophets results but merely provide a wrapper.